portfolio-project-matching-app/components/ProjectCard.js

```diff
@@ -17,32 +17,36 @@ const ProjectCard = ({ project }) => {
                 <p className='text-xl font-medium'>{project.name}</p>
             </div>
             <hr className='w-11/12 sm:w-9/12 border-b-2 border-gray-400'/>
-            <div className='flex flex-wrap'>
+            {/* Commented out, but leaving in just in case we decide to add this back */}
+            {/* <div className='flex flex-wrap'>
                 {project.technologies.map((technology) => {
                     return (
                         <div key={technology.id} className='py-1 pr-2'>
                             <Button text={technology.name}/>
                         </div>
                 )
                 })}
-            </div>
+            </div> */}
             <div className=''>
                 <p>{project.description}</p>
             </div>
-            <div className='mt-4'>
-                <p>{project.census} out of {project.capacity} ({project.capacity - project.census} positions left)</p>
+            <div className='mt-2 flex flex-wrap'>
+                <div className='self-center'>
+                    {project.census} out of {project.capacity} ({project.capacity - project.census} positions left)
+                    </div>
+                <div className='p-1'>
+                    <Button text='join' type='btnGeneral' />
+                </div>
             </div>
             <div className='flex flex-wrap'>
-                {project.users.map((user) => {
+                {/* Commented out, but leaving in just in case we decide to add this back */}
+                {/* {project.users.map((user) => {
                     return (
                     <div key={user.id} className='py-1 pr-2'>
                         <UserIcon imgPath='/../public/user.ico' username={user.username} />
                     </div>
                     )
-                })}
-                <div className='p-1'>
-                    <Button text='join' type='btnGeneral' />
-                </div>
+                })} */}
```

Comment:
Did away with the user listing.

Reply:
I think this makes sense to remove this information from `browseProjects` and only show when someone clicks on "View More Info"